server/index.js

```diff
@@ -8,11 +8,23 @@ const prisma = new PrismaClient();
 app.use(cors());
 app.use(express.json());
 
+//endpoint to get Many
 app.get("/menu_items", async (req, res) => {
   const getFood = await prisma.food.findMany();
   res.json(getFood);
 });
 
+//Add server endpoint to GET a single food item
+app.get("/menu_items", async (req, res) => {
```

Comment:
how do you feel about changing this name to be singular?

```suggestion
app.get("/menu_item", async (req, res) => {
```